### ISSUE TITLE
Problems with Scrollspy on Ipad and Ipad 2 and 3

### ISSUE THREAD

**Issue by filippolcr** [Dogfalo/materialize#2492]
I'm testing my site on Ipad2 and Ipad3 and scrolling is jerky.
I googled and I found that there are problems with jquery $.animate()
I tried to replace $.animate() with velocity:

```
var offset = $(this.hash).offset().top + 1;
$('html, body').animate({ scrollTop: offset /*- 200 */}, {duration: 1000, queue: false, easing: 'easeOutCubic'});
```

```
var offsetV = $(this.hash).offset().top + 1;
$('html, body').velocity( 'scroll' , {duration: 2000, offset: offsetV, easing: 'easeOutCubic'});
```

but nothing improves.

Ok, the problem is that my jquery and velocity knowledge and overall animation is very low and I can 't modify the plugin to optimize performance.

This site works like a charm on Iphone 6, Iphone5, nokia lumia 535, and desktops.

**Comment by acburst** [comment]
I think this is an issue with how the scroll event works on some touch devices. It will only fire a scroll event after the screen has finished moving. At this moment I'm not sure of any good workaround if anyone has any ideas.

**Comment by filippolcr** [comment]
Is Materialize ScrollSpy plugin the same that is implement in Bootstrap Framework?
I make this question because I've recently developed a site with Bootstrap 3 and I doesn't reveal the same behaviour on the same devices.
But maybe is so because in this site, that one  I'm developing with Materialize, I've set a background image and in general  there are many more images and objects to display.
I hope for an improvement with future releases, Materialize is awesome. Thanks
